script.js

```diff
@@ -0,0 +1,647 @@
+
+const title = document.createElement('h1');
+const info = document.createElement('div');
+const tagBody = document.querySelector('body');
+title.setAttribute('class', 'title');
+info.setAttribute('class', 'info');
+tagBody.appendChild(title);
+tagBody.appendChild(info);
+title.innerHTML = 'RSS Виртуальная клавиатура';
+info.innerHTML = 'Клавиатура создана в операционной системе Windows. Для переключения языка комбинация: левый ctrl'
+
+const keyboard = document.createElement('div');
+const keyboardRow1 = document.createElement('div');
+const keyboardRow1Shift = document.createElement('div');
+const keyboardRow1Eng = document.createElement('div');
+const keyboardRow2 = document.createElement('div');
+const keyboardRow2Eng = document.createElement('div');
+const keyboardRow2Caps = document.createElement('div');
+const keyboardRow3 = document.createElement('div');
+const keyboardRow3Eng = document.createElement('div');
+const keyboardRow3Caps = document.createElement('div');
+const keyboardRow4 = document.createElement('div');
+const keyboardRow4Eng = document.createElement('div');
+const keyboardRow4Caps = document.createElement('div');
+const keyboardRow5 = document.createElement('div');
+const textArea = document.createElement('textarea');
+
+textArea.setAttribute('style', 'textarea');
+tagBody.appendChild(textArea);
+
+keyboard.setAttribute('class', 'keyboard');
+tagBody.appendChild(keyboard);
+
+keyboardRow1.setAttribute('class', 'keyboard__row1');
+keyboard.appendChild(keyboardRow1);
+
+keyboardRow1Shift.setAttribute('class', 'keyboard__row1Shift');
+keyboard.appendChild(keyboardRow1Shift);
+
+keyboardRow1Eng.setAttribute('class', 'keyboard__row1Eng');
+keyboard.appendChild(keyboardRow1Eng);
+
+keyboardRow2.setAttribute('class', 'keyboard__row2');
+keyboard.appendChild(keyboardRow2);
+
+keyboardRow2Eng.setAttribute('class', 'keyboard__row2Eng');
+keyboard.appendChild(keyboardRow2Eng);
+
+keyboardRow2Caps.setAttribute('class', 'keyboard__row2Caps');
+keyboard.appendChild(keyboardRow2Caps);
+
+keyboardRow3.setAttribute('class', 'keyboard__row3');
+keyboard.appendChild(keyboardRow3);
+
+keyboardRow3Eng.setAttribute('class', 'keyboard__row3Eng');
+keyboard.appendChild(keyboardRow3Eng);
+
+keyboardRow3Caps.setAttribute('class', 'keyboard__row3Caps');
+keyboard.appendChild(keyboardRow3Caps);
+
+keyboardRow4.setAttribute('class', 'keyboard__row4');
+keyboard.appendChild(keyboardRow4);
+
+keyboardRow4Eng.setAttribute('class', 'keyboard__row4Eng');
+keyboard.appendChild(keyboardRow4Eng);
+
+keyboardRow4Caps.setAttribute('class', 'keyboard__row4Caps');
+keyboard.appendChild(keyboardRow4Caps);
+
+keyboardRow5.setAttribute('class', 'keyboard__row5');
+keyboard.appendChild(keyboardRow5);
+
+
+
+
+
+
+
+//-----------маленькие руссские буквы-------------
```

Comment:
if you take out all the logic you can avoid comments naming this file smallRussianLetters